packages/kinetic-client/kinetic-client.0.0.6/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "1.2"
+name: "kinetic-client"
+version: "0.0.6"
+maintainer: "Romain Slootmaekers <romain.slootmaekers@cloudfounders.com>"
+authors: "Romain Slootmaekers <romain.slootmaekers@cloudfounders.com>"
+homepage: "https://github.com/cloudfounders/kinetic-ocaml-client"
+bug-reports: "https://github.com/cloudfounders/kinetic-ocaml-client"
+dev-repo: "https://github.com/cloudfounders/kinetic-ocaml-client"
```

Comment:
Yes, I will open issues for both these problems. It's certainly not your fault. Thanks for your attention to detail. :-)